src/helics/application_api/Federate.hpp

```diff
@@ -40,12 +40,12 @@ class CloningFilter;
 class Federate
 {
   public:
-    /** the allowable states of the federate*/
-    enum class states : char
+    /** the allowable operation modes of the federate*/
+    enum class modes : char
     {
         startup = 0,  //!< when created the federate is in startup state
-        initialization = 1,  //!< entered after the enterInitializingMode call has returned
-        execution = 2,  //!< entered after the enterExectuationState call has returned
+        initializing = 1,  //!< entered after the enterInitializingMode call has returned
+        executing = 2,  //!< entered after the enterExectuationState call has returned
         finalize = 3,  //!< the federate has finished executing normally final values may be retrieved
```

Comment:
Any reason for this change? To be consistent, maybe it should be `startup`, `initialize`, `execute`, `finalize` and `error`.

Reply:
to be consistent with the function calls `enterInitializingMode`, `enterExecutingMode`